core/src/main/java/com/linecorp/armeria/server/docs/FieldLocation.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright 2019 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.linecorp.armeria.server.docs;
+
+/**
+ * The location of a field.
+ */
+public enum FieldLocation {
+
+    /**
+     * The field is located in the path.
+     */
+    PATH,
+
+    /**
+     * The field is located in the header.
+     */
+    HEADER,
+
+    /**
+     * The field is located in the query.
+     */
+    QUERY,
+
+    /**
+     * The field is located in the request body.
+     */
+    BODY,
+
+    /**
+     * The location of the field is unspecified.
+     */
+    DEFAULT
```

Comment:
If it's unspecified, why not naming it `UNSPECIFIED`?

Reply:
Just follow the rule from `FieldRequirement`. Will change both, Thank you!